persistence-utils-product/product.json

```diff
@@ -12,10 +12,21 @@
           },
           {
             "groupId": "com.axonivy.utils.persistence",
-            "artifactId": "persistence-utils-test",
+            "artifactId": "persistence-utils-demo-tool",
             "version": "${version}",
-            "type": "iar",
-            "importInWorkspace": "false"
+            "type": "iar"
+          },
+          {
+            "groupId": "com.axonivy.utils.persistence",
+            "artifactId": "persistence-utils-demo-test",
+            "version": "${version}",
+            "type": "iar"
+          },
+          {
+            "groupId": "com.axonivy.utils.persistence",
```

Comment:
same for this

Reply:
I would skip deployment this artifact, if not already done and remove it from here.